mongodb/6.2/modules/ROOT/pages/mongodb-connector-reference.adoc

```diff
@@ -68,7 +68,7 @@ Release Notes: xref:release-notes::connector/mongodb-connector-release-notes-mul
 | Password a| String |  The password to use for authentication. If you don't want to use authentication, the password and username must both be empty.  |  |
 | Replica Set Name a| String |  The name of the replica set to which to connect. Use this field to connect to a MongoDB cluster.  |  |
 | Authentication Source a| String |  |  |
-| Compressors a| Array of String |  List of compressors to enable network compression for communication between this client and a mongod/mongos instance. If you specify multiple compressors, then the order in which you list the compressors matter as well as the communication initiator. |  |
+| Compressors a| Array of String |  List of compressors to enable network compression for communication between this client and a mongod or mongos instance. If you specify multiple compressors, then the order in which you list the compressors matter as well as the communication initiator. |  |
```

Comment:
"matters"

It is not clear what this is trying to say in terms of the communication initiator.